data-prepper-plugins/opensearch-source/src/main/java/org/opensearch/dataprepper/plugins/source/OpenSearchSource.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.dataprepper.plugins.source;
```

Comment:
Please change this package to `package org.opensearch.dataprepper.plugins.source.opensearch;`

Reply:
Thanks @graytaylor0  for the review comments. The changes are incorporated.